web/client/components/mapcontrols/scale/ScaleBox.jsx

```diff
@@ -25,7 +25,7 @@ var ScaleBox = React.createClass({
     getDefaultProps() {
         return {
             id: 'mapstore-scalebox',
-            scales: mapUtils.getGoogleMercatorScales(0, 21),
+            scales: mapUtils.getGoogleMercatorScales(0, 28),
```

Comment:
As far as I'm concerned, sure, but it is desired that with the default behaviour the zoom buttons can zoom in further than the available scales in the scalebox? An effect of this is that when one zooms in to max with the buttons, the corresponding scale does not exist as an option in the scalebox combo and consequently the first item of the combo will be set as current, even if it mismatches the actual map scale.